lib/aullar/view.moon

```diff
@@ -793,8 +795,11 @@ View = {
     pos = @position_from_coordinates(event.x, event.y)
     if pos
       @cursor\move_to :pos, extend: true
-    elseif event.y < 0
-      @cursor\up extend: true
+    elseif event.y < 10
```

Comment:
Thanks @nilnor - I've tested and indeed it should indeed be `@margin`.  I felt a bit dirty submitting a PR with a magic number, but I couldn't quite understand what was causing the offset.  I'll fix.